types/index.d.ts

```diff
@@ -317,6 +318,9 @@ export type Admin = {
   fetchTopicOffsets(
     topic: string
   ): Promise<Array<{ partition: number; offset: string; high: string; low: string }>>
+  fetchTopicOffsetsByTimestamp(
+    topic: string, timestamp?: number
+  ): Promise<SeekEntry[]>
```

Comment:
This return type doesn't look right. I would except it to return the same type as `fetchTopicOffsets`.

Reply:
`fetchTopicOffsetsByTimestamp` will not provide the `low` field in the same meaning, so just ignored. I kinda suggest change the return type of `fetchTopicOffsets` from `Promise<Array<{ partition: number; offset: string; high: string; low: string }>>` to `Promise<Array<SeekEntry & { high: string; low: string }>>`, which is more clear.

Reply:
That sounds good!

Reply:
Done. I've squashed all the commits in this PR into one commit. Should be clear to merge.